controllers/testdata/crd/cluster.x-k8s.io_clusters.yaml

```diff
@@ -0,0 +1,1386 @@
+---
```

Comment:
Now that I've added e2e tests, we have a `test/` folder on the project root. Maybe we could move this file to `test/controllers/data` to avoid putting test data in the controller package.

Reply:
Yes, sure, as setup-envtest is very flexible, we can move these files outside the controller data.

Reply:
Folder moved